package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "flickrapi",
-  "version": "0.3.32",
+  "version": "0.3.33",
```

Comment:
leave this as `.32` please, `npm version patch` automatically bumps the version up and creates the git tag